api/app/invitationletter/api.py

```diff
@@ -53,39 +77,34 @@ def post(self):
             passport_expiry_date=passport_expiry_date,
             invitation_letter_sent_at=datetime.now().strftime("%Y-%m-%d"),
             to_date=to_date,
-            from_date=from_date,
+            from_date=from_date
         )
 
-        offer = db.session.query(Offer).filter(
-            Offer.user_id == user_id).first()
-
-        if not offer:
-            return errors.OFFER_NOT_FOUND
-
         invitation_template = None
-
-        if offer.accommodation_award and offer.accepted_accommodation_award \
-                and offer.travel_award and offer.accepted_travel_award:
+        # No offer, but a registration = guest registration - TODO check travel and accom assumed for Invited Guests.
+        if(not offer and registration):
+            invitation_template = db.session.query(InvitationTemplate).filter(
+                InvitationTemplate.event_id == event_id).filter(
+                InvitationTemplate.send_for_both_travel_accommodation).first()
```

Comment:
Lets use the most basic template for invited guests - 99% of them *won't* have travel and accommodation provided by us, we can handle the exceptions manually.

Reply:
Cool, this has been done. I have changed the invited guest's invitation templates to the General template.